src/Mvc/Mvc.Razor/src/Microsoft.AspNetCore.Mvc.Razor.csproj

```diff
@@ -11,6 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="$(SharedSourceRoot)CryptographyAlgorithms\**\*.cs" />
```

Comment:
I'd rather we minimize the number of approaches we're taking to get things working. Please file a follow-up issue to go back to unshared `internal`s with `InternalsVisibleTo`.